ntbs-service/DataMigration/NotificationImportService.cs

```diff
@@ -250,5 +255,57 @@ private async Task<List<string>> FilterOutImportedIdsAsync(PerformContext contex
             }
             return legacyIdsToImport;
         }
+        
+        private async Task MigrateOrUpdateCaseManager(Notification notification,
+            PerformContext context,
+            string requestId)
+        {
+            var legacyNotificationCaseManager =
+                (await _migrationRepository.GetNotificationsById(new List<string> {notification.LegacyId}))
+                .Select(n => n.CaseManager)
+                .SingleOrDefault();
+            if (legacyNotificationCaseManager == null)
+            {
+                return;
+            }
+            var existingCaseManager = await _userRepository.GetUserByUsername(legacyNotificationCaseManager);
+            var ntbsNotificationCaseManager = new User();
+            if (existingCaseManager != null)
+            {
+                ntbsNotificationCaseManager = existingCaseManager;
+            }
+            else
+            {
+                ntbsNotificationCaseManager.IsActive = false;
+                ntbsNotificationCaseManager.Username = legacyNotificationCaseManager;
+            }
+
+            var legacyUser = (await _migrationRepository.GetLegacyUserByUsername(legacyNotificationCaseManager)).First();
+            ntbsNotificationCaseManager.GivenName = legacyUser.GivenName;
+            ntbsNotificationCaseManager.FamilyName = legacyUser.FamilyName;
+            ntbsNotificationCaseManager.DisplayName = ntbsNotificationCaseManager.GivenName + " " + ntbsNotificationCaseManager.FamilyName;
```

Comment:
This looks like logic that would better belong on the User class. Also, string interpolation is generally preferable to concatenation for readability.

Reply:
Are you just referring to the display name logic?

Reply:
Yes